lib/services/channel.js

```diff
@@ -0,0 +1,43 @@
+var Service = require('./service');
+
+/**
+ * Service for interacting with the channel endpoints on the Beam REST API.
+ * @access public
+ * @augments {Service}
+ */
+function ChannelService () {
+    Service.apply(this, arguments);
+}
+
+ChannelService.prototype = new Service();
```

Comment:
`Object.create(Service.prototype);` if something different, imo. Don't need to force browser bundles to import `util` if they aren't already.